PWGHF/Tasks/taskXicc.cxx

```diff
@@ -118,26 +118,24 @@ struct HfTaskXiccMc {
   void init(o2::framework::InitContext&)
   {
     auto vbins = (std::vector<double>)bins;
-    registry.add("hmassSig", "#Xi^{++}_{cc} (rec. matched) candidates;inv. mass (p K #pi) (GeV/#it{c}^{2});entries", {HistType::kTH2F, {{400, 3.2, 4.0}, {vbins, "#it{p}_{T} (GeV/#it{c})"}}});
-    registry.add("hmassBg", "#Xi^{++}_{cc} (rec. unmatched) candidates;inv. mass (p K #pi) (GeV/#it{c}^{2});entries", {HistType::kTH2F, {{400, 3.2, 4.0}, {vbins, "#it{p}_{T} (GeV/#it{c})"}}});
+    registry.add("hmassSig", "#Xi^{++}_{cc} (rec. matched) candidates;inv. mass (p K #pi #pi) (GeV/#it{c}^{2});entries", {HistType::kTH2F, {{400, 3.2, 4.0}, {vbins, "#it{p}_{T} (GeV/#it{c})"}}});
+    registry.add("hmassBg", "#Xi^{++}_{cc} (rec. unmatched) candidates;inv. mass (p K #pi #pi) (GeV/#it{c}^{2});entries", {HistType::kTH2F, {{400, 3.2, 4.0}, {vbins, "#it{p}_{T} (GeV/#it{c})"}}});
```

Comment:
I noticed that your histograms don't follow the naming conventions. Please fix them. Have a look at the instructions for developers for further details on the MC names.

Reply:
Hi Vit, I followed the naming conventions in the new commit. I am sorry to bother you. Would you please let me know if I have to rename the histograms?

Reply:
The ones you renamed look fine. Can you please fix the names of the rest? I see only 4 histograms left.

Reply:
The histograms are: `hptcand`, `hptprong0`, `hptprong1`, `hselectionstatus`.

Reply:
Hi @vkucera, they are the histograms related to the data. Should I also add a `Rec` in the name of the histogram?

Reply:
> Hi @vkucera, they are the histograms related to the data. Should I also add a `Rec` in the name of the histogram?

No, of course not. I meant the capitalisation. :-)